src/test/jtx/TrustedPublisherServer.h

```diff
@@ -170,29 +176,77 @@ class TrustedPublisherServer
         auto const manifest = makeManifestString(
             publisherPublic_, publisherSecret_, keys.first, keys.second, 1);
 
-        std::string data = "{\"sequence\":" + std::to_string(sequence) +
-            ",\"expiration\":" +
-            std::to_string(expiration.time_since_epoch().count()) +
-            ",\"validators\":[";
+        std::vector<std::pair<std::string, std::string>> blobInfo;
```

Comment:
Consider a struct instead of `pair<string,string>`. Pairs where both elements have the same type are error prone.

Reply:
Done